transcode/transcode.go

```diff
@@ -76,18 +76,6 @@ func RunTranscodeProcess(transcodeRequest TranscodeSegmentRequest, streamName st
 	}
 	// Go back to the root directory to set as the output for transcode renditions
 	targetTranscodedPath := path.Dir(path.Dir(segmentedOutputManifestURL.Path))
-	// Use the same manifest filename that was used for the segmented manifest
-	targetTranscodedManifestFilename := path.Base(segmentedOutputManifestURL.String())
-	// Generate the new output path of the transcoded manifest
-	targetTranscodedOutputPath := path.Join(targetTranscodedPath, targetTranscodedManifestFilename)
-	// Generate the manifest output URL from the manifest output path (e.g. s3+https://USER:PASS@storage.googleapis.com/user/hls/index.m3u8)
-	tpath, err := url.Parse(targetTranscodedOutputPath)
-	if err != nil {
-		return outputs, fmt.Errorf("failed to parse targetTranscodedOutputPath: %s", err)
-	}
-	targetTranscodedOutputURL := segmentedOutputManifestURL.ResolveReference(tpath)
-	fmt.Println(targetTranscodedOutputURL)
-	// Generate the rendition output URL (e.g. s3+https://USER:PASS@storage.googleapis.com/user/hls/)
 	tout, err := url.Parse(targetTranscodedPath)
```

Comment:
Don't we need the above deleted code to generate the filenames?

Reply:
targetTranscodedOutputURL is just being printed though and not used, which is why I could delete it

Reply:
ooh yeah i left that there for a follow up PR to change the rendition folder structures. I'll add it back when I put up that PR.